File: sdkwork-ai-sdk-go/types/run_step_list_response.go
package types

// OpenAI run step list response
type RunStepListResponse struct {
	Object string `json:"object"`
	Data []RunStepResponse `json:"data"`
	HasMore bool `json:"has_more"`
	FirstId string `json:"first_id"`
	LastId string `json:"last_id"`
}

File: sdkwork-ai-sdk-typescript/src/types/run-step-list-response.ts
import type { RunStepResponse } from './run-step-response';

/** OpenAI run step list response */
export interface RunStepListResponse {
  /** Object type */
  object?: string;
  /** Run step list */
  data?: RunStepResponse[];
  /** Whether more data exists */
  has_more?: boolean;
  /** First item id */
  first_id?: string;
  /** Last item id */
  last_id?: string;
}
